examples/bootstrap/Makefile

```diff
@@ -0,0 +1,29 @@
+ifndef UUID
+UUID := $(shell uuidgen | tr '[:upper:]' '[:lower:]')
+endif
+RG := GHCI-$(shell echo ${UUID} | cut -d '-' -f 1,5)
+STORAGE := ghci$(shell echo ${UUID} | cut -d '-' -f 5)
+TF_PARAMS := -var-file=example.tfvars -var resource_group_name=${RG} -var inbound_files={"inbound_files/authcodes.sample"="license/authcodes","inbound_files/init-cfg.sample.txt"="config/init-cfg.txt"} -var obew_files={"obew_files/authcodes.sample"="license/authcodes","obew_files/init-cfg.sample.txt"="config/init-cfg.txt"} -var storage_account_name=${STORAGE}
+
+init: 
+	@../../makefile.sh init
+
+validate:
+	@../../makefile.sh validate
+
+plan: init
+	@../../makefile.sh plan '${TF_PARAMS}'
+
+plan_file: init
+	@../../makefile.sh plan_file '${TF_PARAMS}'
+
+apply_file:
+	@../../makefile.sh apply_file
+
+apply: init plan_file apply_file
+
+indepotency:
+	@../../makefile.sh indepotency '${TF_PARAMS}'
+
+destroy:
+	@../../makefile.sh destroy '${RG}'
```

Comment:
I was thinking of how to deduplicate this part since we have it in all Makefiles.
Maybe we can create one Makefile with all that options and we can pass it through 
`make -f ../Makefile TF_PARAMS='${TF_PARAMS}'` do you thing this is achievable?

Reply:
achievable, but not for the moment. Having this duplicated gives you flexibility when moving module by module from plain commands to `terratest`